draft-ietf-anima-brski-prm.md

```diff
@@ -928,99 +945,106 @@ The fields contained in the `pledgevoucherrequesttrigger` are:
+The Registrar-Agent MUST sign the `agent-signed-data` member using its EE credential (which must correspond to an LDevID certificate signed by the domain owner to be able to assert agent-proximity).
+The JWS Signature is generated over the JWS Protected Header and the JWS Payload as described in {{Section 5.1 of RFC7515}}.
 
-The ietf-voucher-request:voucher data is extended with two additional parameters that MUST be included:
 
-* `agent-provided-proximity-registrar-cert`: base64-encoded registrar EE certificate (provided in tPVR by the Registrar-Agent); enables the registrar to verify that it is the desired registrar for handling the PVR
+### Response Artifact: Pledge Voucher-Request (PVR) {#pvr-artifact}
 
-* `agent-signed-data`: base64-encoded agent-signed-data (provided in tPVR by the Registrar-Agent); enables the registrar to verify and log, which Registrar-Agent was in contact with the pledge, when verifying the PVR
+The Pledge Voucher-Request (PVR) artifact SHALL be an authenticated self-contained object signed by the pledge, containing an extended Voucher-Request artifact based on {{!RFC8995}}.
+The BRSKI-PRM related enhancements of the `ietf-voucher-request` YANG module are defined in {{I-D.ietf-anima-rfc8366bis}}.
 
-The enhancements of the YANG module for the ietf-voucher-request with these new leaves are defined in {{I-D.ietf-anima-rfc8366bis}}.
-
-The PVR is signed using the pledge's IDevID credential contained as x5c parameter of the JOSE header.
+For the JWS-signed JSON format used by this specification, the PVR artifact MUST be a JWS Voucher structure as defined in {{!I-D.ietf-anima-jws-voucher}}, which MUST contain the JSON PVR Data defined in {{pvr-data}} as JWS Payload.
+{{pvr_representation}} summarizes the serialization of the JWS-signed JSON PVR artifact:
 
 ~~~~
-# The PVR in General JWS Serialization syntax
 {
-  "payload": BASE64URL(UTF8(ietf-voucher-request:voucher)),
+  "payload": BASE64URL(UTF8(JSON PVR Data)),
   "signatures": [
     {
       "protected": BASE64URL(UTF8(JWS Protected Header)),
       "signature": BASE64URL(JWS Signature)
     }
   ]
 }
+~~~~
+{: #pvr_representation title='PVR Representation in General JWS JSON Serialization Syntax' artwork-align="left"}
 
-# Example: Decoded Payload "ietf-voucher-request:voucher"
-  representation in JSON syntax
+#### JSON PVR Data {#pvr-data}
+
+The JSON PVR Data MUST contain the following fields of the `ietf-voucher-request` YANG module as defined in {{I-D.ietf-anima-rfc8366bis}};
+note that this makes optional leaves in the YANG definition mandatory for the PVR artifact:
+
+* `created-on`: SHALL contain the current date and time as standard date/time string as defined in {{Section 5.6 of !RFC3339}};
+  if the pledge does not have synchronized time, it SHALL use the `created-on` value from the BRSKI-PRM Agent-Signed Data, received in the trigger to create the PVR
```

Comment:
Proposal to add here that the time should be advanced as also note further down: 
if the pledge does not have synchronized time, it SHALL use the `created-on` value from the BRSKI-PRM Agent-Signed Data, received in the trigger to create the PVR, advanced by the local clock.

Reply:
Harmonized all `created-on` statements with same advance clause for pledges without RTC.

Reply:
Thank you